### ISSUE TITLE
train with the pre-trained model but start from begining

### ISSUE THREAD

**Issue by Bananajia** [davidsandberg/facenet#231]
Hi,

I don't know if my parameter is right:

facenet_train_classifier.py --logs_base_dir ~/facenet_train/logs/facenet --gpu_memory_fraction 0.5 --models_base_dir ~/facenet_train/models/facenet/ --data_dir ~/datasets/casia/casia_maxpy_mtcnnpy_182 --image_size 160 --model_def models.inception_resnet_v1 --lfw_dir ~/datasets/lfw/lfw_mtcnnpy_160 --optimizer RMSPROP --learning_rate -1 --max_nrof_epochs 500 --keep_probability 0.8 --random_crop --random_flip --learning_rate_schedule_file ~/Downloads/facenet-master/data/learning_rate_schedule_classifier_casia.txt --weight_decay 5e-5 --center_loss_factor 1e-4 --center_loss_alfa 0.9 --lfw_pairs ~/Downloads/facenet-master/data/pairs.txt --lfw_batch_size 100 --pretrained_model ~/facenet_train/models/facenet/20170331-115906/model-20170331-115906.ckpt-29000

and the model model-20170331-115906.ckpt-29000 was trained with the para:
facenet_train_classifier.py --logs_base_dir ~/facenet_train/logs/facenet --gpu_memory_fraction 0.5 --models_base_dir ~/facenet_train/models/facenet/ --data_dir ~/datasets/casia/casia_maxpy_mtcnnpy_182 --image_size 160 --model_def models.inception_resnet_v1 --lfw_dir ~/datasets/lfw/lfw_mtcnnpy_160 --optimizer RMSPROP --learning_rate -1 --max_nrof_epochs 500 --keep_probability 0.8 --random_crop --random_flip --learning_rate_schedule_file ~/Downloads/facenet-master/data/learning_rate_schedule_classifier_casia.txt --weight_decay 5e-5 --center_loss_factor 1e-4 --center_loss_alfa 0.9 --lfw_pairs ~/Downloads/facenet-master/data/pairs.txt --lfw_batch_size 100

and after 500 epochs,  
499000	0.98267	0.94767
500000	0.98250	0.93833

but when I use this pre-trained model to continue training, it showed:
0	        0.96633	0.77100
1000	0.97067	0.77967
2000	0.97067	0.84500

means it started training from the begining?

**Comment by Bananajia** [comment]
the pre-trained model is ~/facenet_train/models/facenet/20170331-181641/model-20170331-181641.ckpt-500000

**Comment by Bananajia** [comment]
The AC does not change so much, but in LFW, it seems to have low accuracy

**Comment by 1senlin** [comment]
hi , could you  please send me a pre-trained model , -20170331-181641.ckpt-50000,or you can share me a link to donload 
thanks,
